examples/interaction/tools/tool_context_menu.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
```

Comment:
Suggest adding a docstring and example metadata. This would be a nice gallery addition with a thumbail showing part of a context menu

Reply:
Sure, start an issue and submit a PR.